src/kanalytics.js

```diff
@@ -132,12 +133,12 @@ export default class KAnalytics extends BasePlugin {
    */
   _sendSeekAnalytic(): void {
     let now = new Date().getTime();
-    if (this._lastSeekEvent === 0 || this._lastSeekEvent + SEEK_OFFSET < now) {
+    if (this._lastSeekEvent + SEEK_OFFSET < now && (this.player.config.type !== LIVE || this.player.config.dvr)) {
```

Comment:
what is `this._lastSeekEvent + SEEK_OFFSET < now`? not intuitive from code? also, add brackets for complex if statements, e.g. `((a===x) && (b===y))`

Reply:
this logic for avoid seeking while scrubbing.
we store the time of last sent seek event and compare this value + a threshold named SEEK_OFFSET equals 2000 with the `now` value. with this logic we send `seek` event only after 2000 sec from the previews.